api/src/main/resources/cb_search_api.yaml

```diff
@@ -239,6 +241,13 @@ definitions:
           for leaves in the tree. Parents don't have domain ids and concept
           id will be used in the case that a parent is selectable.
         type: string
+      attribute:
+        description: specifies if extra attributes are needed
```

Comment:
What does this mean? How does this relate to predefinedAttributes?

Reply:
This flag signals to the UI that this criteria will be using a form to collect attribute(operator and operands) data from the user.

Reply:
OK. You might rename this to "attributesRequired" or something.

Reply:
Renamed to hasAttributes